cns/client/client_test.go

```diff
@@ -391,3 +392,39 @@ func TestCNSClientDebugAPI(t *testing.T) {
 	t.Logf("PodIPConfigState: %+v", inmemory.HTTPRestServiceData.PodIPConfigState)
 	t.Logf("IPAMPoolMonitor: %+v", inmemory.HTTPRestServiceData.IPAMPoolMonitor)
 }
+
+func TestNew(t *testing.T) {
+	emptyRoutes, _ := buildRoutes(defaultBaseURL, clientPaths)
+	tests := []struct {
```

Comment:
Will more cases be added? Otherwise, are table tests needed?

Reply:
more tests to follow, just need to get this base refactor out